src/main/kotlin/commands/ForceUpdateCommand.kt

```diff
@@ -0,0 +1,30 @@
+package commands
+
+import Bot
+import Command
+import doesLater
+
+object ForceUpdateCommand : Command("forceupdate"){
+    init{
+        doesLater{
+            try{
+                // TODO: Check member permission
+                if( message.author!!.id != 563138570953687061){ return@doesLater }
```

Comment:
Change this to `if (!message.hasPermission(FORCE_UPDATE) return@doesLater`, create FORCE_UPDATE in the permissions manager